src/containers/shared/components/TransactionTable/TransactionTableRow.tsx

```diff
@@ -64,3 +64,7 @@ export const TransactionTableRow = ({ tx, hasTokensColumn = false }: Props) => {
     </li>
   )
 }
+
+TransactionTableRow.defaultProps = {
```

Comment:
Export the props to prevent this from being required.

Reply:
done(?)